tests/Functional/LocalVariableNamingSniffFunctionalTest.php

```
@@ -85,6 +90,149 @@ public function testAttributedPropertiesAreNotFlagged(): void {
+   * - Comments separated by code are left unchanged.
+   * - Trailing comments on previous statements are left unchanged.
+   */
+  public function testPhpcbfFixesCommentVariableReferences(): void {
+    $fixed = $this->runPhpcbf(
+      static::$fixtures . DIRECTORY_SEPARATOR . 'CommentVariableNaming.php',
+      'DrevOps.NamingConventions.LocalVariableNaming'
+    );
+
+    // Check if phpcbf actually applied fixes. PHPCS 3 does not support
+    // auto-fixing for this sniff, so skip fix-specific assertions.
+    $fixes_applied = str_contains($fixed, '$module_handler = get_service(');
```

PHPMD (2.15.0)

[error] The variable $fixes_applied is not named in camelCase. (undefined)

(CamelCaseVariableName)